js/components/table/table.component.js

```diff
@@ -0,0 +1,89 @@
+import React, {Component} from 'react';
+import {repositories} from '../../mocks';
+import HighchartRepo from '../highchart/highchart.component.js';
+import {Table, TableBody, TableHeader, TableRow, TableHeaderColumn, TableRowColumn} from 'material-ui';
+
+export default class RepositoriesTable extends Component {
+
+  constructor(...props) {
+    super(...props);
+
+    this.state = ({
+      repositories,
+      selectedRows: [],
+      reposToCompare: []
+    });
+  }
+
+
+  handleRowSelection(selectedRows) {
+    this.repositoriesList.map((item, index)=>  {
+      return selectedRows.map(i=> {if(i===index) {
+        this.state.reposToCompare[index] = item
+      }})
+    })
+  }
+
+  get repositoriesList() {
+    const itemsCount = 5;
+    return this.state.repositories.slice(0, itemsCount)
+  }
+
+  render () {
+    // pass data this.state.reposToCompare
+    const chartOptions = {
```

Comment:
you must not put any const, function, etc in render. Better move it to some method